quickstart-semantic-search/README.md

```diff
@@ -1,3 +1,15 @@
+---
+page_type: sample
+languages:
+  - csharp
+name: C# semantic search with Azure AI Search
```

Comment:
semantic search is now semantic ranker or semantic ranking.  I'd go with "ranking" for line 5